include/openspace/scene/profile.h

```diff
@@ -164,20 +163,19 @@ class Profile {
      * \return The Lua library that contains all Lua functions available for profiles
      */
     static scripting::LuaLibrary luaLibrary();
+
+    std::string file_subName_assets = "";
+    std::string assetFileExtension = ".asset";
```

Comment:
This variable is only used in the OpenSpaceEngine::loadInitAssetSection and as described above, the function is (rightfully) not called anymore, so this variable can die, too

Reply:
Also gone